src/main/java/us/abstracta/wiresham/ParallelPacketStep.java

```diff
@@ -0,0 +1,61 @@
+package us.abstracta.wiresham;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class ParallelPacketStep extends PacketStep {
```

Comment:
Now looking at it, PacketStep should probably be a sub class of FlowStep. Because semantically parallelStep is not a "packet" step, but a multi packet step (as is includeStep)?